playbook-website/app/javascript/site_styles/_site-style.scss

```diff
@@ -16,6 +16,14 @@ body {
   }
 }
 
+.anchorjs-link {
+  font-size: 2em !important;
+  vertical-align: middle !important;
+  position: relative;
+  top: -3px;
+}
```

Comment:
Can this be moved inside our parent class? Lets align to tokens and remove and the negative. Can we not use a kit to do all this?